### PR TITLE
Add new port: Descent 3

### PR DESCRIPTION
**Pull request by JeodC**
# New Port for Descent 3

## Game Information
- **Title**: Descent 3

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

### CODE REVIEW

File: ports/descent3/gameinfo.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version='1.0' encoding='utf-8'?>
+<gameList>
+  <game>
+    <path>./Descent 3.sh</path>
+    <name>Descent III</name>
+    <desc>Play the exciting final installment in the Descent trilogy! Saved by amateur do-gooders, the Material Defender sets out to stop the robot virus once and for all! Pilot through 3D space, navigating through familiar indoor mines and outdoor open space terrain. Uncover the PTMC's involvement behind the mystery and battle mercenaries hired to prevent your success.</desc>
+    <image>./descent/cover.png</image>
```
Comment (Slayer366) [inline ports/descent3/gameinfo.xml:7]:
```suggestion
    <image>./descent3/cover.jpg</image>
```

**Review by Slayer366 — CHANGES_REQUESTED** [review]

File: ports/descent3/cover.jpg
Comment (Cebion) [inline ports/descent3/cover.jpg:1]: Cover has to be the same format as the screenshots. 640x480.
Its not a literal cover from boxart but a screenshot with embedded logo for example.
This art is shown in emulationstation. A screenshot with mixed art is also okay. Like skraper v3 screenshots.

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/descent3/cover.jpg
Comment (JeodC) [reply ports/descent3/cover.jpg:1]: This is from skraper actually.

File: ports/descent3/gameinfo.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version='1.0' encoding='utf-8'?>
+<gameList>
+  <game>
+    <path>./Descent 3.sh</path>
+    <name>Descent III</name>
+    <desc>Play the exciting final installment in the Descent trilogy! Saved by amateur do-gooders, the Material Defender sets out to stop the robot virus once and for all! Pilot through 3D space, navigating through familiar indoor mines and outdoor open space terrain. Uncover the PTMC's involvement behind the mystery and battle mercenaries hired to prevent your success.</desc>
+    <image>./descent3/cover.jpg</image>
```
Comment (Slayer366) [inline ports/descent3/gameinfo.xml:7]: Per Cebion's suggestion:
```suggestion
    <image>./descent3/screenshot.png</image>
```

**Review by Slayer366 — CHANGES_REQUESTED** [review]

**Pull request merged**